src/circleci/rollcage/core.clj

```diff
@@ -167,17 +170,22 @@
    level  :- String
    exception :- Throwable
    url :- (s/maybe String)
-   custom :- (s/maybe {s/Any s/Any})]
+   custom :- (s/maybe {s/Any s/Any})
+   & [person :- {:id String
```

Comment:
`make-rollbar` is only called in one place, and it's private, so we should be able to make the new argument a required parameter, right?